vue/src/components/Shared.vue

```diff
@@ -85,13 +99,13 @@ export default {
           "vars": {"poolSize": 10000, ... defaultVarsWithoutHorizontalScaling, "experimentName": "horizontalScaling" }
         },
         "excludeSnapshots": this.excludeSnapshots || false,
-        "filterRuns": "Latest"
+        "filterRuns": "Latest",
```

Comment:
I'll let them through this time, but generally good to put formatting changes in different PRs.  Tools like IntelliJ will let you go through change-by-change deciding what to include in the commit.